schemas/file.yml

```diff
@@ -225,3 +225,17 @@
 
       short: A fork is additional data associated with a filesystem object.
       example: Zone.Identifer
+
+    - name: origin_referrer_url
+      level: extended
+      type: keyword
+      ignore_above: 8192
+      description: The url of the webpage that linked to the file.
+      example: https://example.com
+
+    - name: origin_url
+      level: extended
+      type: keyword
+      ignore_above: 8192
+      description: The url where the file is hosted.
```

Comment:
Have you considered redirects? For example, when using CDNs, the initial URL can be redirected to various download URLs. Is that meaningful for your use case? Should the description specify if this is the initial request or a redirected URL?

Reply:
Yes, I have implemented this field (`origin_url`) to retrieve the actual URL from which the file was downloaded using the [Mark of the Web (MoTW) information](https://en.wikipedia.org/wiki/Mark_of_the_Web) in Windows.  (MoTW is metadata added to files downloaded from the internet in Windows. )

For example, if you download `ejabberd_20.12-0_amd64.deb` from https://www.process-one.net/downloads/downloads-action.php?file=/20.12/ejabberd_20.12-0_amd64.deb, the download is redirected, and the file is actually retrieved from https://static.process-one.net/ejabberd/downloads/20.12/ejabberd_20.12-0_amd64.deb.

In MoTW, as shown in the attached image, the redirect URL is saved as the file's host URL.

<img width="825" alt="image" src="https://github.com/user-attachments/assets/bd7c371f-d9bf-4109-8328-43e1e84eecce">